packages/components/src/components/form/FractionButton/FractionButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button, Tooltip } from '@trezor/components';
+
+import { ButtonVariant } from '../../buttons/buttonStyleUtils';
+
+const DEFAULT_VARIANT = 'tertiary';
+
+export type FractionButtonProps = {
+    label: string;
```

Comment:
let's remove label, I think it is not needed. Everything can be translatable. In English, it can be `10%` in different language it can be `10 %` or `1/10` in Korean. I dont know